Platform/AlderlakeBoardPkg/Library/Stage2BoardInitLib/AcpiPlatform.c

```diff
@@ -271,7 +271,9 @@ PlatformUpdateAcpiTable (
   VOID                        *FspHobList;
   PLATFORM_DATA               *PlatformData;
   FEATURES_CFG_DATA           *FeaturesCfgData;
+#if FixedPcdGet8 (PcdTccEnabled)
   TCC_CFG_DATA                *TccCfgData;
+#endif
```

Comment:
If CfgData_Tcc.yaml file is not used in CfgDataDef.yaml, then when PcdTccEnabled is set to TRUE it could cause build failure.  Will you remove TCC_ENABLE from boardconfig.py?

Reply:
Yes, we will remove TCC_ENABLE from that platform-specific boardconfig.py where TCC is not supported.

Reply:
got it.